raiden-ts/src/raiden.ts

```diff
@@ -399,13 +408,14 @@ export class Raiden {
         },
       ],
     );
+    const cleanConfig = config && decode(PartialRaidenConfig, omitBy(config, isUndefined));
```

Comment:
Pretty neat code. Sometimes `lodash` chains are quite useful. :grimacing: 
Please close this conversation after reading.